File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.py
s, k, h = map(int, input().split())

sum = s+k+h

if sum >= 100:
    print("OK")

else:
    min1 = min(s, k)
    min2 = min(min1, h)

    if min2 == s:
        print("Soongsil")
    elif min2 == k:
        print("Korea")
    elif min2 == h:
        print("Hanyang")

File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = sc.nextLine().split(" ");
        int s = Integer.parseInt(str[0]);
        int k = Integer.parseInt(str[1]);
        int h = Integer.parseInt(str[2]);

        int sum = s + k + h;

        if (sum >= 100)
            System.out.println("OK");
        else
        {
            int min1 = Math.min(s, k);

            int min2 = Math.min(min1, h);

            if (min2 == s)
                System.out.println("Soongsil");

            else if (min2 == k)
                System.out.println("Korea");

            else if (min2 == h)
                System.out.println("Hanyang");
        }
    }
}